app/views/layouts/wpcc/engine.html.erb

```diff
@@ -6,7 +6,18 @@
 <% end %>
 
 <% content_for(:engine_content) do %>
-  <%= yield %>
+  <div class="l-constrained">
+    <div class="l-container-tool">
+      <h1><%= t('wpcc.title') %></h1>
+
+      <div class="wpcc">
```

Comment:
Can we move the ``<h1>`` above into this container? It will make it easier if we need to apply styles, we could follow BEM (e.g. ``<h1 class="wpcc__title">``)